tests/plot_sync/test_sender.py

```diff
@@ -107,3 +107,13 @@ def new_response_message(sync_id: int, message_id: int, message_type: ProtocolMe
     # Test invalid message-type
     sender._response = new_expected_response(3, 0, ProtocolMessageTypes.plot_sync_start)
     assert not sender.set_response(new_response_message(3, 0, ProtocolMessageTypes.plot_sync_loaded))
+
+
+def test_sync_done_with_negative_duration_does_not_crash(bt: BlockTools) -> None:
+    sender = Sender(bt.plot_manager, HarvestingMode.CPU)
+    sender.sync_start(0, True)
+
+    try:
+        sender.sync_done([], -1)
+    except Exception as e:
+        assert False, f"'sender.sync_done' raised an exception: {e}"
```

Comment:
You can remove this try/except handler. pytest does this automatically - just call `sync_done` - if it raises, pytest will fail the test automatically for you

Reply:
perfect 👍